spec/puppetserver/ca/cli_spec.rb

```diff
@@ -57,4 +57,17 @@
       'sign',
       /.*Usage.* puppetserver ca sign.*Display this command-specific help output.*/m
   end
+
+  #This test is not specific for clean, but just an action to validate our test
+  #would work with other options as well.
```

Comment:
(nit) we usually put a space between the `#` and the words
```suggestion
  # This test is not specific for clean, but just an action to validate our test
  # would work with other options as well.
```

Reply:
Justin and I worked through that test yesterday. I manually tested by replacing clean with other action like revoke, list, generate, etc... I ended up picking clean just for a representation.  Thanks for spotting the extra space btw 😄